locale/tl/LC_MESSAGES/django.po

```diff
@@ -14,92 +14,153 @@ msgstr ""
 "X-Generator: Pootle 2.5.1.1\n"
 "X-POOTLE-MTIME: 1518452107.934725\n"
 
-#: gem/constants.py:7
+#: gem/constants.py:6 gem/constants.py:9
+msgid "male"
+msgstr "Lalaki"
+
+#: gem/constants.py:7 gem/constants.py:10
 msgid "female"
 msgstr "Babae"
-
-#: gem/constants.py:8
+#: gem/constants.py:8 gem/constants.py11
 msgid "don't want to answer"
 msgstr "Ayokong sagutin"
 
-#: gem/forms.py:81
-#: gem/forms.py:209
-#: temp/temp_ve/profiles/forms.py:122
-#: temp/temp_ve/profiles/forms.py:259
-#: temp/temp_ve/profiles/forms.py:303
+#: gem/forms.py:35
+msgid ""
+"Sorry, but that is an invalid display name. Please don't use your email "
```

Comment:
@KaitCrawford @saeedmb8 shouldn't each of these translations have the templates they are used in at the top?